test/utils/react.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+
+export function scryRenderedDOMComponentsWithId(tree, id) {
+    return TestUtils.findAllInRenderedTree(tree, function(inst) {
+        return TestUtils.isDOMComponent(inst) && inst.getAttribute('id') === id;
+    });
+}
+
+export function scryRenderedDOMComponentsWithAttribute(tree, attr, value = '') {
+    return TestUtils.findAllInRenderedTree(tree, function(inst) {
+        return TestUtils.isDOMComponent(inst) && inst.getAttribute(attr) === value;
+    });
+}
+
+export function findRenderedDOMComponentsWithId(tree, id) {
```

Comment:
Doesn't this already exist and isn't it called `TestUtils.findRenderedDOMComponentWithId` ?